src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseData/IntellisenseData.cs

```diff
@@ -163,7 +163,7 @@ internal virtual bool DoesNameCollide(string name)
         /// <summary>
         /// Should unqualified enums be suggested.
         /// </summary>
-        internal virtual bool SuggestUnqualifiedEnums => true;
+        internal virtual bool SuggestUnqualifiedEnums => false;
```

Comment:
There may need to be some flag here just to ensure that we don't break Canvas Apps.

Reply:
nit, can this be a 'private set'? Ie, we want to treat this as a readonly property and not let others set it. 

Or just => 
Binding.NameResolver.SuggestUnqualifiedEnums;